.changeset/great-nails-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'graphql-ws': patch
+---
+
+adds a crossws adapter for compatibility with all environments supported by crossws
```

Comment:
The text will apear in the changelog. It's good practice to make it appealing.
```suggestion
[crossws](https://crossws.h3.dev/) adapter for compatibility with all supported environments
```

Also maybe add an example of how to use in the changelog itself, leaving an empty new line after the title will have everything below be the content of the changelog's entry.